### PR TITLE
Update swiftproxy to use service override

### PR DESCRIPTION
**Pull request by stuggi**
Removes creation of routes. Those get done in the openstack-operator. Via service overrides the openstack-operator customize the services for the endpoints to the desired state specified in the ctlplane CR using externalEndpoints or the service override in the service template.

Jira: [OSP-26690](https://issues.redhat.com//browse/OSP-26690)

Depends-On: https://github.com/openstack-k8s-operators/lib-common/pull/313

### CODE REVIEW

File: controllers/swiftproxy_controller.go [hunk 1]
```diff
@@ -122,35 +122,89 @@ func (r *SwiftProxyReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	serviceLabels := swiftproxy.Labels()
 
 	// Create a Service and endpoints for the proxy
-	var swiftPorts = map[endpoint.Endpoint]endpoint.Data{
-		endpoint.EndpointAdmin: endpoint.Data{
+	var swiftPorts = map[service.Endpoint]endpoint.Data{
+		service.EndpointAdmin: {
```
Comment (stuggi) [inline controllers/swiftproxy_controller.go:126]: we do not create admin endpoint for other services, is the admin endpoint required for swift?

Comment (cschwede) [reply controllers/swiftproxy_controller.go]: Good point - no, it's no longer required, this should work fine without it.

**Review by cschwede — COMMENTED** [review]
LGTM, will run another test after lib-common#313 is merged.

**Pull request closed without merging**